src/server/models/UserProfile.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const userProfileSchema = new mongoose.Schema({
+    user_id: {
+        type: Number,
+        required: true,
```

Comment:
Would the `user_id` in this schema reference the one from the user schema?
I'm not sure where we can connect both of them together.

Reply:
Yeah, I'm not entirely sure either. What would we use the user_id for ? Where/when would it come into play ?